www/content/lexique.md

```diff
@@ -93,25 +93,46 @@ de la technologie du clavier pour développer des dispositions de clavier.
 
 [Couche]{.a name="couche-def"} (<i lang="en">layer</i>)
 
-: TODO
+: Abstraction pour pouvoir disposer plusieurs caractères sur une même touche.
+Exemple le plus courant de couches : en tapant des lettres sur la couche « alpha »
+on obtient les lettres en minuscules, en gardant la touche [shift]{.kbd} enfoncée
+on obtient les lettres en majuscules, c’est donc 2 couches différentes. Ce concept
+de couches permet de diminuer le nombre de touches nécessaires sur un clavier et
+est utilisé à son plein potentiel sur les petits claviers ergonomiques pour diminuer
+la distance que les doigts ont à parcourir.
 
 Touche morte (<i lang="en">dead key</i>)
 
-: TODO
+: Touche de clavier qui ne produit aucun résultat lorsqu’elle est enfoncée puis relâchée,
+mais modifie le comportement de la prochaine touche qui sera enfoncée.
```

Comment:
Suggestion d’ajout : 
> Une touche morte est généralement utilisée pour produire des diacritiques.

Reply:
Bonne suggestion, c'est plus concret. "Produire des _lettres_ accentuées" est peut-être plus précis. On produitdes caractères composés, par juste la diacritique.

Reply:
Le verbe diacritiser existe également, mais pas sûr que cela rende les choses plus claires.

Reply:
J’ai utilisé le mot « diacritiques » plutôt que « lettres accentuées » pour faire la réf à une autre définition mais c’est peut-être inutile ou pédant. Ou alors on peut avoir les deux (« pour produire des lettres accentuées ou autres diacritiques »).

Reply:
> pour produire des lettres accentuées ou autres diacritiques

J’aime bien !

Si l’on veut être pédant, les accents sont une famille de diacritiques. Je ne trouve pas d’explication sur Wikipedia, mais la page anglaise donne :
> accents (so called because the acute, grave, and circumflex were originally used to indicate different types of [pitch accents](https://en.wikipedia.org/wiki/Pitch_accent) in the [polytonic transcription](https://en.wikipedia.org/wiki/Polytonic_transcription) of [Greek](https://en.wikipedia.org/wiki/Greek_language))

Ta proposition est cohérente et on peut juste ajouter un lien Wikipedia, cf. suggestion de changement.